Flow.Launcher/ViewModel/SettingWindowViewModel.cs

```diff
@@ -382,6 +382,33 @@ public List<ColorScheme> ColorSchemes
             }
         }
 
+
+
+        public class LauncherPosition
+        {
+            public string Display { get; set; }
+            public LauncherPositions Value { get; set; }
+        }
+
+        public List<LauncherPosition> LauncherPositions
```

Comment:
This is clever, we may use this structure for all the combobox. Probably we want to cache this result? But that may break some behavior when changing language.

Reply:
I just copied and pasted the existing structure. 😋